eng/pipelines/runtime-linker-tests.yml

```diff
@@ -0,0 +1,33 @@
+# The purpose of this pipeline is to exercise local developer workflow in the consolidated
+# runtime repo. In particular, it is supposed to run the root "build" script just like any
+# normal developer normally would and monitor regressions w.r.t. this fundamental scenario.
+
+trigger: none
```

Comment:
I think this should run on CI batched as well to catch potential issues where there is a merge conflict that breaks a change.

Reply:
sounds good, I'll turn on CI then.